shipyard-controller/handler/sequencedispatcher.go

```diff
@@ -80,7 +81,13 @@ func (sd *SequenceDispatcher) Add(queueItem models.QueueItem) error {
 		//if there are multiple shipyard we should only write
 		return sd.add(queueItem)
 	}
+}
 
+func (sd *SequenceDispatcher) addItemToQueue(queueItem models.QueueItem) error {
+	if err2 := sd.add(queueItem); err2 != nil {
+		return err2
+	}
+	return ErrSequenceBlockedWaiting
```

Comment:
returning `ErrSequenceBlockedWaiting` seems counterintuitive at this point, as adding an item to the queue usually is already a consequence of the sequence being blocked - can we move the return of the `ErrSequenceBlockedWaiting` to the caller of this method?

Reply:
Sure, we can move it there, but that will mean one more `if err != nil { return err} return ErrSequenceBlockedWaiting` in each caller place, so than it makes no sense to have a function for this few lines and we can paste them on the caller place directly